src/components/ScholarshipForm.jsx

```diff
@@ -229,6 +230,10 @@ function ScholarshipForm({ scholarship, submitFn, onSubmitError }) {
     </Grid>
   );
 
+  stepperItems.Review = (
+    <ScholarshipDetailCard scholarship={{ data: formik.values }} review />
```

Comment:
Add some margin above it? You can surround this with a Box and apply styles directly on the Box.